lace/src/interface/engine/data.rs

```diff
@@ -1734,18 +1723,14 @@ mod tests {
         let result =
             incr_category_in_codebook(&mut codebook, &suppl_metadata, 0, 3);
 
-        assert!(result.is_ok());
+        // assert!(result.is_ok());
+        result.unwrap();
```

Comment:
I think I prefer the `assert` rather than `unwrap`-ping the result and discarding the value